src/isomorphic/classic/types/__tests__/ReactPropTypesProduction-test.js

```diff
@@ -23,7 +23,8 @@ describe('ReactPropTypesProduction', function() {
     __DEV__ = false;
     oldProcess = process;
     global.process = {
-      env: Object.assign({}, process.env, {NODE_ENV: 'production'}),
+      cwd: process.cwd,
```

Comment:
Don't know reason why, but on Windows when `global.process` has replaced it loses access to `process.cwd`, test runner setup fails to use those function.